target_chains/solana/sdk/js/solana_utils/src/transaction.ts

```diff
@@ -307,16 +298,9 @@ export class TransactionBuilder {
             })
           );
         }
-        if (
-          args.jitoTipLamports &&
-          index % jitoBundleSize === jitoBundleSize - 1
-        ) {
+        if (args.jitoTipLamports && index % jitoBundleSize === 0) {
```

Comment:
The old logic of putting the jito tip in the last transaction of the bundle was bad. Because the last bundle would not tip if not full.